core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -791,7 +791,7 @@ portfolio.pending.supportTicketOpenedByPeer=Your trading peer opened a support t
 portfolio.pending.noReceiverAddressDefined=No receiver address defined
 portfolio.pending.removeFailedTrade=If the mediator or arbitrator could not close that trade you can move it yourself \
```

Comment:
There's no "failed trades" panel right, as far as I know...just a "history" panel (like it is now). 

Is that still the case?

Regardless, this text sounds overly complicated to me. Suggestion:

```
portfolio.pending.removeFailedTrade=Is this a failed trade? If so, would you like to manually \
close it, so that it no longer shows as an open trade?
```

Reply:
> There's no "failed trades" panel right now, as far as I know...just a "history" panel that shows successful trades and failed trades.

There are failed trades still in the code at least 😉 It is a separate tab next to the history tab.
<img width="1312" alt="Bildschirmfoto 2019-10-10 um 13 40 45" src="https://user-images.githubusercontent.com/170962/66565784-9bd18500-eb63-11e9-81d5-d00a7b964932.png">
You can trigger this popup in the pending trades screen with `ALT+Y`
I'll adapt the text accordingly.
![Bildschirmfoto 2019-10-10 um 13 43 49](https://user-images.githubusercontent.com/170962/66565951-0256a300-eb64-11e9-9948-e8cb1198919c.png)
![Bildschirmfoto 2019-10-10 um 13 44 05](https://user-images.githubusercontent.com/170962/66565967-0be00b00-eb64-11e9-85f0-2fd23be66a74.png)